finetune/lora.py

```diff
@@ -250,7 +250,7 @@ def train(
             save_lora_checkpoint(fabric, model, checkpoint_path)
 
 
-@torch.inference_mode()
+@torch.no_grad()
```

Comment:
Since validation (as a sanity check) is the first step of the training, the `lora_ind` property will be called first here. So if the validation is running in inference_mode, the indices will be also stored in an inference tensor. That explains the issue above.